### ISSUE TITLE
Metadata: Contains alt text

### ISSUE THREAD

**Issue by nicolegaines** [daisy/ebraille#116]
Should there be an optional or required metadata element to capture whether or not the eBraille file includes alt text for graphics? This option is provided in schema.org: meta name="schema:accessibilityFeature" content="alternativeText"

**Comment by wfreeman-aph** [comment]
I think alt text should be a requirement of the standard and so I don't think we need an alt text metadata element as it would be redundant with whether the document contains graphics.

**Comment by wfreeman-aph** [comment]
It was agreed in the 11/14/24 meeting to close this ticket as not planned.

**Comment by egli** [comment]
My understanding at the meeting was that since alt text will be mandated by the spec, there is no need for some metadata to state that yes, this file does contain alt text.
